package.json

```diff
@@ -20,20 +20,26 @@
     "ios",
     "android"
   ],
-  "author": "Tomas Roos <ptomasroos@gmail.com>",
+  "authors": [
+    "Tomas Roos <ptomasroos@gmail.com>",
+    "Joshua Conlin <joshjconlin@yahoo.com>"
+  ],
   "license": "MIT",
-  "peerDependencies": {
-     "react-native": "*",
-     "react": "*"
-   },
-   "devDependencies": {
-     "babel-eslint": "7.2.3",
-     "eslint": "3.19.0",
-     "eslint-config-airbnb": "14.1.0",
-     "eslint-plugin-import": "2.2.0",
-     "eslint-plugin-jsx-a11y": "4.0.0",
-     "eslint-plugin-react": "6.10.3"
-   },
+  "dependencies": {
+    "prop-types": "^15.5.10",
+    "react": "^15.5.4",
+    "react-native": "^0.44.0",
+    "react-native-vector-icons": "^4.1.1",
+    "immutable": "^3.8.1"
```

Comment:
my thought about it was that in larger applications where all data is immutable and particularly in use with redux-form/immutable it makes it easy to just stick the slider in as a form component. since its returning an array, not a single digit, some conversion would be needed otherwise. I'm open to discussion on it though certainly